packages/rollup-plugin-import-meta-assets/src/rollup-plugin-import-meta-assets.js

```diff
@@ -62,19 +73,96 @@ function importMetaAssets({ include, exclude, warnOnError, transform } = {}) {
 
   return {
     name: 'rollup-plugin-import-meta-assets',
-
     async transform(code, id) {
       if (!filter(id)) {
         return null;
       }
 
-      const ast = this.parse(code);
-      const magicString = new MagicString(code);
+      // Part 1: resolve dynamic template literal expressions
+      const parsed = this.parse(code);
+
+      let dynamicURLIndex = -1;
+      let ms;
+
+      await asyncWalk(parsed, {
+        enter: async node => {
+          const importMetaUrlType = getImportMetaUrlType(node);
+
+          if (importMetaUrlType !== 'dynamic') {
+            return;
+          }
+          dynamicURLIndex += 1;
+
+          try {
+            // see if this is a Template Literal with expressions inside, and generate a glob expression
+            const glob = dynamicURLToGlob(node.arguments[0], code.substring(node.start, node.end));
+
+            if (!glob) {
+              // this was not a variable dynamic url
+              return;
+            }
+
+            const { globby } = await import('globby');
+            // execute the glob
+            const result = await globby(glob, { cwd: path.dirname(id) });
+            const paths = result.map(r =>
+              r.startsWith('./') || r.startsWith('../') ? r : `./${r}`,
+            );
```

Comment:
I guess we gotta double check here in the test results if the returned paths have POSIX separators on Windows. I guess this ran green in CI earlier using `fast-glob` so it should be all good.

Reply:
Joren @jorenbroekema is on Windows, I assume he tested this?

Reply:
Looks like the tests are running on Windows so it's probably fine 🤷🏻

Reply:
they do and they fail due to timeouts quite often, so it's hard to make them all run and turn green
I restarted 3-4 times yesterday, no luck, not sure why so unstable though
so I won't say it's 100% working, but in one of the runs I saw lots of tests passing

Reply:
Yup the flaky tests are in `browser-logs` I think so if all the other tests ran green then we are all good.